README.md

```diff
@@ -1 +1,58 @@
-# lib-System.Security.Cryptography
\ No newline at end of file
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=nanoframework_System.Security.Cryptography&metric=alert_status)](https://sonarcloud.io/dashboard?id=nanoframework_System.Security.Cryptography) [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=nanoframework_System.Security.Cryptography&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=nanoframework_System.Security.Cryptography) [![NuGet](https://img.shields.io/nuget/dt/nanoFramework.System.Security.Cryptography.svg?label=NuGet&style=flat&logo=nuget)](https://www.nuget.org/packages/nanoFramework.System.Security.Cryptography/) [![#yourfirstpr](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://github.com/nanoframework/Home/blob/main/CONTRIBUTING.md) [![Discord](https://img.shields.io/discord/478725473862549535.svg?logo=discord&logoColor=white&label=Discord&color=7289DA)](https://discord.gg/gCyBu8T)
+
+![nanoFramework logo](https://raw.githubusercontent.com/nanoframework/Home/main/resources/logo/nanoFramework-repo-logo.png)
+
+-----
+
+### Welcome to the .NET **nanoFramework** System.Security.Cryptography Library repository
```

Comment:
```suggestion
# Welcome to the .NET **nanoFramework** System.Security.Cryptography Library repository
```

Reply:
Actually, we have a mix of these two styles throughout our repos... 😯 
Let's lock on one and adjust as needed. 
I like the `###` because it brings the highlight that this is meant to provide visually and the `#` makes it too big and maybe distractive.
But I'm not adamant on any of them. Let's agree on the style, OK?

Reply:
The first `#`is the main title of the doc and should be like this so it can be properly processed as the main title :-)